### ISSUE TITLE
Exception when attempting to use TTS in editing cards

### ISSUE THREAD

**Issue by kiba** [AwesomeTTS/awesometts-anki-addon#118]
```Add-ons possibly involved: ⁨AwesomeTTS - Add speech to your flashcards⁩

Caught exception:
Traceback (most recent call last):
  File "aqt/webview.py", line 37, in cmd
  File "aqt/webview.py", line 123, in _onCmd
  File "aqt/webview.py", line 544, in _onBridgeCmd
  File "aqt/editor.py", line 412, in onBridgeCmd
  File "/home/kiba/.local/share/Anki2/addons21/1436550454/awesometts/gui/common.py", line 126, in _show
    self._instance.show()
  File "/home/kiba/.local/share/Anki2/addons21/1436550454/awesometts/gui/generator.py", line 729, in show
    super(EditorGenerator, self).show(*args, **kwargs)
  File "/home/kiba/.local/share/Anki2/addons21/1436550454/awesometts/gui/base.py", line 404, in show
    dropdown = self.findChild(QtWidgets.QComboBox, 'service')
RuntimeError: wrapped C/C++ object of type EditorGenerator has been deleted```

This happens pretty often when I need to edit a card. Adding a card seems to work fine for the most part.

**Comment by lucwastiaux** [comment]
seems to be a similar issue to #102

**Comment by lucwastiaux** [comment]
I need someone to record a screen capture showing this bug. I'm unable to reproduce it.

**Comment by lucwastiaux** [comment]
Looks like this is a potential reproduction:

1. Add a note from the browser using Notes -> Add Note
2. Click the AwesomeTTS button within the Add Note dialog, and add a sound
3. Close the Add window using the close button on the window
4. Go to another card
5. Try to click the AwesomeTTS button to add a sound.

this generates the following error:

> Error
> An error occurred. Please start Anki while holding down the shift key, which will temporarily disable the add-ons you have installed.
> If the issue only occurs when add-ons are enabled, please use the Tools > Add-ons menu item to disable some add-ons and restart Anki, repeating until you discover the add-on that is causing the problem.
> When you've discovered the add-on that is causing the problem, please report the issue on the add-on support site.
> Debug info:
> Anki 2.1.26 (70784154) Python 3.8.1 Qt 5.14.1 PyQt 5.14.1
> Platform: Linux
> Flags: frz=True ao=True sv=1
> Add-ons, last update check: 2020-08-27 10:27:44
> Add-ons possibly involved: ⁨awesometts-anki-addon⁩
> 
> Caught exception:
> Traceback (most recent call last):
>   File "aqt/webview.py", line 34, in cmd
>   File "aqt/webview.py", line 114, in _onCmd
>   File "aqt/webview.py", line 518, in _onBridgeCmd
>   File "aqt/editor.py", line 408, in onBridgeCmd
>   File "/home/luc/.local/share/Anki2/addons21/awesometts-anki-addon/awesometts/gui/common.py", line 126, in _show
>     self._instance.show()
>   File "/home/luc/.local/share/Anki2/addons21/awesometts-anki-addon/awesometts/gui/generator.py", line 729, in show
>     super(EditorGenerator, self).show(*args, **kwargs)
>   File "/home/luc/.local/share/Anki2/addons21/awesometts-anki-addon/awesometts/gui/base.py", line 404, in show
>     dropdown = self.findChild(QtWidgets.QComboBox, 'service')
> RuntimeError: wrapped C/C++ object of type EditorGenerator has been deleted
>

**Comment by lucwastiaux** [comment]
Notes for myself: seems to be the third case described here: http://enki-editor.org/2014/08/23/Pyqt_mem_mgmt.html , **C++ object is deleted by Qt but Python wrapper still exists**

**Comment by lucwastiaux** [comment]
@kiba  I expect this issue is fixed in v1.21, just released. please check.
